vault/vault.go

```diff
@@ -37,11 +39,19 @@ func NewDefaultVault() EnvVault {
 	return EnvVault{client: vaultClient}
 }
 
-// DecryptAllEnv decrypts all envs that contains a Vault path.
-// All values staring with `vault://` are overridden by the secret valued stored in the path.
-// For instances:
-//  Input: ["db_url=url","db_pass=vault://secret/db_pass"]
-// Output: ["db_url=url","db_pass=ACTUAL_SECRET_PASS"]
+// DecryptAllEnv decrypts all env vars that contain a Vault path.  All values
+// staring with `vault://` are overridden by the secret valued stored in the
```

Comment:
`valued` -> `value`

Reply:
Good catch! It turns out it's in there twice like that. Fixed.